service-base/src/main/java/org/eclipse/hono/service/amqp/AmqpServiceBase.java

```diff
@@ -672,13 +672,15 @@ protected void handleSessionOpen(final ProtonConnection con, final ProtonSession
     }
 
     /**
-     * Is called whenever a proton connection was closed. The implementation is intentionally empty.
+     * Is called whenever a proton connection was closed.
      * <p>
-     * Subclasses should override this method to publish this as an event on the vertx bus if desired.
+     * Subclasses may override this method to publish this as an event on the vertx bus if desired. If they choose to
```

Comment:
In order to know if I should override this method, it would be helpful to know, what the default implementation actually does